test/setupTests.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Enzyme, { shallow, render, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+// React 16 Enzyme adapter
+Enzyme.configure({ adapter: new Adapter() });
+// Make Enzyme functions available in all test files without importing
+global.React = React;
```

Comment:
I could get away without needing to add these globals. Can you refer to this: https://github.com/MadaraUchiha-314/yet-another-react-starter-code

Reply:
Modified the test cases similar to ones used in https://github.com/MadaraUchiha-314/yet-another-react-starter-code. 
Let me know if we need any changes